src/select/src/components/CheckboxSelect.js

```diff
@@ -21,7 +21,7 @@ export const defaultProps = {
 const CheckboxSelect = (props) => (
   <SelectController
       {...props} // eslint-disable-line indent
-      render={((selectProps) => <ReactSelect {...mergeDeep(defaultProps, selectProps)} />)} />
+      render={((selectProps) => <ReactSelect {...mergeDeep(selectProps, defaultProps)} />)} />
```

Comment:
Doesn't this change make user provided props useless? I feel like this is def gonna break something.

Reply:
You mean `components.DropdownIndicator = DropdownIndicator;`?

Reply:
Test pass...

Reply:
Lemme take a deeper look.

Reply:
The intended functionality should be that if someone needs to make additional changes to the any of the LUK Selects, they would be extending the LUK defaults, rather than extending the react-select defaults.

Reply:
In this example...
```
<CheckboxSelect
    ...
    styles={{
      option: (base) => ({
        ...base,
        color: magenta
      })
    }} />
```

... both `mergeDeep(defaultProps, selectProps)` and `mergeDeep(selectProps, defaultProps)` behave the same.